templates/telemetry/clients_last_seen_v1/view.sql

```diff
@@ -28,3 +29,28 @@ SELECT
   sample_id AS _sample_id_string
 FROM
   `moz-fx-data-derived-datasets.telemetry_derived.clients_last_seen_v1`
+WHERE
+  submission_date < DATE '2019-11-14'
+UNION ALL
+SELECT
+  -- We cannot use UDFs in a view, so we paste the body of udf_bitpos(bits) literally here.
+  CAST(SAFE.LOG(days_seen_bits & -days_seen_bits, 2) AS INT64) AS days_since_seen,
+  CAST(SAFE.LOG(days_visited_5_uri_bits & -days_visited_5_uri_bits, 2) AS INT64) AS days_since_visited_5_uri,
+  CAST(SAFE.LOG(days_opened_dev_tools_bits & -days_opened_dev_tools_bits, 2) AS INT64) AS days_since_opened_dev_tools,
+  CAST(SAFE.LOG(days_created_profile_bits & -days_created_profile_bits, 2) AS INT64) AS days_since_created_profile,
```

Comment:
I'd prefer that we do the union in a CTE and then add these days_since_ columns on top of the CTE to avoid duplicating the logic between the two sides of the union.